app/src/main/kotlin/at/bitfire/davdroid/db/WebDavDocumentDao.kt

```diff
@@ -21,8 +23,15 @@ interface WebDavDocumentDao {
     @Query("SELECT * FROM webdav_document WHERE mountId=:mountId AND (parentId=:parentId OR (parentId IS NULL AND :parentId IS NULL)) AND name=:name")
     fun getByParentAndName(mountId: Long, parentId: Long?, name: String): WebDavDocument?
 
-    @Query("SELECT * FROM webdav_document WHERE parentId=:parentId")
-    fun getChildren(parentId: Long): List<WebDavDocument>
+    @RawQuery
+    fun getChildren(query: RoomRawQuery): List<WebDavDocument>
```

Comment:
It can be used to run other queries than for children, too. So I think it should be called `query()` or something like that.

Also maybe keep it private? No public caller should need it.

Reply:
Renamed, but it cannot be private inside of the interface